packages/twenty-server/src/database/typeorm-seeds/workspace/connectedAccount.ts

```diff
@@ -0,0 +1,64 @@
+import { DataSource } from 'typeorm';
+
+const tableName = 'connectedAccount';
+
+export const seedConnectedAccount = async (
+  workspaceDataSource: DataSource,
+  schemaName: string,
+) => {
+  await workspaceDataSource
+    .createQueryBuilder()
+    .insert()
+    .into(`${schemaName}.${tableName}`, [
+      'id',
+      'createdAt',
+      'updatedAt',
+      'deletedAt',
+      'lastSyncHistoryId',
+      'accountOwnerId',
+      'refreshToken',
+      'accessToken',
+      'provider',
+      'handle',
+    ])
+    .orIgnore()
+    .values([
+    {
+      id: '6773281b-9ac0-4390-9a1a-ab4d2c4e1bb7',
```

Comment:
Seed ids should start with `20202020` so you can do:
```suggestion
      id: '20202020-9ac0-4390-9a1a-ab4d2c4e1bb7',
```